pom.xml

```
@@ -9,7 +9,7 @@
   </parent>
 
   <artifactId>gitlab-plugin</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>1.9.10-SNAPSHOT</version>
```

```suggestion
  <version>${revision}${changelist}</version>
```
Please undo this change